### ISSUE TITLE
mol2 to pdb: parmed vs cpptraj

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#696]
I am not sure if there is an existing issue about mol2 (there are several ones) and I am not sure about the title

pytraj/cpptraj
-----------------

```python
pt.load("pro4.mol2").save("test2.pdb")
```

```bash
ATOM      1  H1  MOL     1       0.588   1.627   1.974  1.00  0.00           H
ATOM      2  C1  MOL     1       0.631   2.240   1.083  1.00  0.00           C
ATOM      3  H2  MOL     1       1.621   2.133   0.653  1.00  0.00           H
```

parmed
----------

```python
pmd.load_file("pro4.mol2").save("test.pdb")
```

```bash
HETATM    1 H1   MOL     1       0.588   1.627   1.974  0.00  0.00          EP  
HETATM    2 C1   MOL     1       0.631   2.240   1.083  0.00  0.00          EP  
HETATM    3 H2   MOL     1       1.621   2.133   0.653  0.00  0.00          EP  
```

so the differences are

- ATOM vs HETATM
- atom element vs `EP`

File: `amber_web/antechamber/pro4.mol2`

**Comment by hainm** [comment]
uhm, `EP`

from ParmEd doc.

```
Contains all of the elements in the periodic table and dictionaries that
store all of the data found in the periodic table for that element, indexed
by the element's symbol. For consistency with AMBER, a fictitious element
'EP' is added to all of the arrays that is just an Extra Point, with no mass
or any other meaningful attribute. It's just a container to put an extra 
charge 
```

I see.

**Comment by hainm** [comment]
probably this issue overlap with guessing atomic number from other issues.

**Comment by hainm** [comment]
related PR: https://github.com/ParmEd/ParmEd/pull/695

**Comment by swails** [comment]
Yes, this is caused by the mol2 file not assigning atomic numbers by default (and therefore winding up with the default "0").

**Comment by swails** [comment]
Fixed by #695
